tinygrad/graph.py

```diff
@@ -10,7 +12,8 @@
 from tinygrad.ops import DeviceBuffer, UnaryOps, BinaryOps, ReduceOps, MovementOps, LoadOps, FusedOps, Op, OpType, LazyOp, get_buffers, get_lazyops
 from tinygrad.helpers import getenv, DEBUG
 
-GRAPH, PRUNEGRAPH, GRAPHPATH = getenv("GRAPH", 0), getenv("PRUNEGRAPH", 0), getenv("GRAPHPATH", "/tmp/net")
+GRAPH, PRUNEGRAPH = getenv("GRAPH", 0), getenv("PRUNEGRAPH", 0)
+GRAPHPATH = getenv("GRAPHPATH", "/tmp/net" if platform.system() != "Windows" else os.path.join(tempfile.gettempdir(), "net"))
```

Comment:
I put this on another line for readability, not sure best way to write to cleanly and limited the num lines...

Reply:
That's fine.